src_multispec/multispec_functions.H

```diff
@@ -267,25 +267,25 @@ void GetMolTot(const GpuArray<Real, MAX_SPECIES>& W,
 
 void ComputeMolconcMolmtotLocal( int nspecies_in,
                 GpuArray<Real, MAX_SPECIES>& molmass_in,
-                Array<Real, MAX_SPECIES>& rho_in,
+                GpuArray<Real, MAX_SPECIES>& RhoN,
                 Real rhotot_in,
-                Array<Real, MAX_SPECIES>& molarconc_in,
+                GpuArray<Real, MAX_SPECIES>& MolarConcN,
                 Real molmtot_in){
 
     GpuArray<Real, MAX_SPECIES> W;
 
     // calculate mass fraction and total molar mass (1/m=Sum(w_i/m_i))
 
     for (int n=0; n<nspecies_in; ++n){
-        W[n] = rho_in[n] / rhotot_in;
+        W[n] = RhoN[n] / rhotot_in;
     }
 
-    GetMolTot(W, molmtot_in);
+    GetMolTot(W, molmtot_in); 
```

Comment:
```suggestion
    GetMolTot(W, molmtot_in);
```
just because I want to show you how cool github can be :)